src/api/handlers/PetFindOneByIdHandler.ts

```diff
@@ -8,8 +8,8 @@ class PetFindOneByIdHandler {
         this.service = new PetService();
     }
 
-    public execute() {
-        const pet = this.service.findOneById();
+    public execute(event: any) {
```

Comment:
- @Aristevao here you are not receiving an `event` object because on line 27 you are passing `event.pathParameters.id`.
- Please update to:
```typescript
class PetFindOneByIdHandler {
    private service: PetService;

    constructor() {
        this.service = new PetService();
    }

    public execute(event: any) {
        const pet = this.service.findOneById(event.pathParameters.id); // here I pass the id
        const response = {
            statusCode: 200,
            headers: {
                "x-custom-header": "My Header Value",
                "Access-Control-Allow-Origin": "*", // Required for CORS support to work
                "Access-Control-Allow-Credentials": true, // Required for cookies, authorization headers with HTTPS
            },
            body: JSON.stringify({ pet }),
        };
        return response;
    }
}

export const handler: Handler = async (event, context, callback) => {
    const response = new PetFindOneByIdHandler().execute(event); // I pass here the event object
    callback(null, response);
};
```
- Don't just copy and paste the code. To learning, it's better you manually write the code...